e2e/tests/list.js

```diff
@@ -11,7 +11,7 @@ describe('List Page', () => {
         'http://localhost:8083/#/comments'
     )(driver);
 
-    before(async () => await ListPagePosts.navigate());
+    beforeEach(async () => await ListPagePosts.navigate());
```

Comment:
All tests were dependent, causing some troubles when adding an `.only` and revealing some issues.

Reply:
Yes and that's expected actually. Some tests are relying on the previous